zstacklib/zstacklib/gpu/vendors/haiguang.py

```
@@ -0,0 +1,56 @@
+    def get_basic_info_cmd(cls, is_windows=False):
+        return "hy-smi -a --json"
+
+    @classmethod
+    def parse_basic_info(cls, output):
+        gpu_infos = []
+        if not output:
+            return gpu_infos
+            
+        try:
+            data = json.loads(output)
+            for card_name, card_data in data.items():
```

Ruff (0.14.14)

Loop control variable `card_name` not used within loop body

Rename unused `card_name` to `_card_name`

(B007)